BuildForSDG/Team-191-GetFunding
# Ft registration

## Description
Added the endpoints to handle registration of the clients, both lenders and borrowers.
Dependencies are stated in the pyproject.toml file, 

Fixes #9 

## How Has This Been Tested?
- Run `python wsgi` to test the changes. (Still trying to learn poetry run command)
- To test run `python3 -m unittest tests.test_register`  in the root of the project.
- **Make sure you have set the DATABASE_URI,SDG_CONFIG and SECRET environment variables.**

## Checklist:
- [x ] I have performed a self-review of my own code
